### PR TITLE
Update product page feature flag values

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update product page feature flag values and adapt the current state based on existing values
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/31381
| Sponsor company   | ~
| How to test?      | - When upgrading from 8.0.0 and product page V2 is enabled for single shop after the upgrade the product page should be enabled<br>- When upgrading from 8.0.0 and product page V2 is enabled for multi shop after the upgrade the product page should be enabled<br>- When upgrading from 8.0.0 and product page V2 is NOT enabled for single shop nor multi shop after the upgrade the product page should be disabled<br><br>In all three cases when you go to Experimental features, there is only one feature flag left for product page its name is now `New product page` and it's a stable feature so it's in the New features block

### CODE REVIEW

**Review by mflasquin — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Comment by HanaRebaiQA** [comment]
Hello @jolelievre 

Thanks for your PR!
Your PR will fix this issue [#31381](https://github.com/PrestaShop/PrestaShop/issues/31381) ?

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @jolelievre 

I have checked your PR. The improvement is NOK.

Here is my checked list : 

- Upgrade from **PS8.0.1 to PS8.1.0 (From create release .zip)** when multi shop enabled : **NOK** :x: 
- Upgrade from **PS8.0.1 to PS8.1.0 (From create release .zip)** when single shop enabled : **NOK** :x: (Same error)
- Upgrade from **PS8.0.1 to PS8.1.0 (From create release .zip)** when multi shop and single shop disabled : **NOK** :x:  (Same error)

**==>** After the upgrade is unsuccessful, the 8.1.0 shop is displayed but the Experimental features is incorrect.

image


https://user-images.githubusercontent.com/92912932/220649259-7a52a5e0-4c10-4b9c-b10c-43ac523944f3.mp4



- Upgrade from **PS8.0.1 to PS8.1.0_beta** when multi shop enabled : **NOK** :x: (Same error)
- Upgrade from **PS8.0.1 to PS8.1.0_beta** when single shop enabled : **NOK** :x:  (Same error)
- Upgrade from **PS8.0.1 to PS8.1.0_beta**  when multi shop and single shop disabled : **NOK** :x: (Same error)

image

- Upgrade from **PS1788 to PS8.1.0 (From create release .zip)** when multi shop and single shop disabled: **NOK** :x: 

image

Could you please check it ?

Thanks!

File: upgrade/sql/8.1.0.sql [hunk 1]
```diff
@@ -28,9 +28,11 @@ CREATE TABLE `PREFIX_product_attribute_lang` (
   `available_later` varchar(255) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
   PRIMARY KEY (`id_product_attribute`, `id_lang`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
-
 /* PHP:ps_810_add_product_attribute_lang_data(); */;
 
+/* Update feature flags */
+/* PHP:ps_810_update_product_page_feature_flags(); */
```
Comment (kpodemski) [inline upgrade/sql/8.1.0.sql:34]:
```suggestion
/* PHP:ps_810_update_product_page_feature_flags(); */;
```

**Comment by kpodemski** [comment]
ping @jolelievre

**Review by kpodemski — APPROVED** [review]

**Review by djoelleuch — APPROVED** [review]
Hello @jolelievre 

I have checked your PR.

The upgrade from PS8.0.1 to PS8.1.0 when multi shop enabled: OK :heavy_check_mark: 
The upgrade from PS8.0.1 to PS8.1.0  when single shop enabled : OK :heavy_check_mark: 
The upgrade from PS8.0.1 to PS8.1.0  when multi shop and single shop disabled : OK :heavy_check_mark: 
The upgrade from PS1788 to PS8.1.0 (From create release .zip) when multi shop and single shop disabled : OK  :heavy_check_mark: 
 the rollback from PS8.1.0 to PS8.0.1 : OK :heavy_check_mark: 
 the rollback from PS8.1.0 to PS1.7.8.8 : OK :heavy_check_mark: 

So, it's QA approved ! 

Thank You :rocket: !

**Comment by kpodemski** [comment]
thanks @jolelievre @djoelleuch

**Pull request merged**